cloudnet/src/main/java/de/dytanic/cloudnet/CloudNet.java

```diff
@@ -231,6 +232,7 @@ public synchronized void start() throws Exception {
 
         this.currentNetworkClusterNodeInfoSnapshot = this.createClusterNodeInfoSnapshot();
         this.lastNetworkClusterNodeInfoSnapshot = this.currentNetworkClusterNodeInfoSnapshot;
```

Comment:
Maybe remove this.current/lastNetworkClusterNodeInfoSnapshot and return the ones out of the current NodeServer in their getters so that no synchronization errors may occur?